package/upgrade/upgrade_manifests.sh

```diff
@@ -1300,6 +1302,23 @@ wait_for_deployment() {
   done
 }
 
+fleet_agent_timestamp(){
+  wait_for_deployment cattle-fleet-local-system fleet-agent
```

Comment:
Maybe we need to suppress the output of calling this `wait_for_deployment` function. Otherwise, the output log lines and the timestamp integer will be saved together in the variables used later for an integer comparison `-ge`.

```
/usr/local/bin/upgrade_manifests.sh: line 1314: [: waiting: integer expression expected
```